library/core/src/fmt/builders.rs

```diff
@@ -1,4 +1,4 @@
-use crate::fmt;
+use crate::fmt::{self, Debug, Formatter};
```

Comment:
You'll need `allow(unused_imports)` since these are only used for intra-doc links.